api/controller/plan.js

```diff
@@ -34,6 +38,69 @@ class PlanController extends Controller {
         });
     }
 
+    publicBrowse(req) { 
+        const columns = [
+            "id",
+            "PLAN_COUNTY_NAME",
+            "PL_NUMBER",
+            "PL_NAME",
+            "PLAN_CHARACTOR_NAME",
+            "geom"
+        ];
+
+        const { query } = req;
+        const response = {
+            "type": "FeatureCollection",
+            "bbox":[],
+            "features": []
+        }
+
+        if (!query.polygon) {
+           throw new Exception.BadRequest('Missing polygon param')
+        }
+
+        const points = req.query.polygon.split(';').map(i=>i.split(',').map(i=>parseFloat(i)));
+        const geojson = {
+            type: "Polygon",
+            coordinates: [points]
+        };
+        if (!GJV.valid(geojson)){
+            throw new Exception.BadRequest("polygon is invalid" );
+        }
+        const polygon = wkt.convert(geojson);
+        const whereRaw = [
+            Knex.raw(`ST_Intersects(geom, ST_GeomFromText("${polygon}",4326))`)
+        ];
+        const order = '-id';
+
+        return super
+            .browse(req, {
+                columns,
+                whereRaw,
+                order,
+                pageSize:1000
+            })
+            .then(rows => {
+                rows.map(row => {
```

Comment:
would either do something like - 
response.features = rows.map(row...

or change the map function to forEach